tests/elite_test.py

```
@@ -121,13 +119,13 @@ def test_set_sdef(self):
     def test_initialize_elite(self):
         inp = deepcopy(self.testInput)
         excel_name = "E-Lite_dummy_Block_Structure_Summary.xlsx"
-        assert inp._sectors_L0_cells_names == {}
-        assert inp._Elite_Input__initialized == False
-        inp._initialize_elite(resources_elite.joinpath(excel_name), check_Elite=True)
-        assert inp._Elite_Input__initialized == True
-        assert 800 not in inp._sectors_L0_cells_names[1]
-        assert 451001 in inp._sectors_L0_cells_names[4]
-        assert 435001 in inp._sectors_L0_cells_names["2 & 3"]
+        assert inp._sectors_L0_cells_names == {}  # type: ignore
+        assert inp._Elite_Input__initialized == False  # type: ignore
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `False`; use `not inp._Elite_Input__initialized:` for false checks

Replace with `not inp._Elite_Input__initialized`

(E712)